src/Microsoft.Data.SqlClient/netcore/src/Microsoft/Data/SqlClient/TdsParser.cs

```diff
@@ -12525,8 +12536,9 @@ internal int ReadPlpUnicodeChars(ref char[] buff, int offst, int len, TdsParserS
         // requested length is -1 or larger than the actual length of data. First call to this method
         //  should be preceeded by a call to ReadPlpLength or ReadDataLength.
         // Returns the actual chars read.
-        internal bool TryReadPlpUnicodeChars(ref char[] buff, int offst, int len, TdsParserStateObject stateObj, out int totalCharsRead)
+        internal bool TryReadPlpUnicodeChars(ref char[] buff, int offst, int len, TdsParserStateObject stateObj, out int totalCharsRead, bool supportRentedBuff, ref bool rentedBuff)
         {
+            bool inputWasRentedBuff = rentedBuff;
```

Comment:
Looks like this is not used?
```suggestion
```

Reply:
Yes, this got moved into the loop. Removed.